### ISSUE TITLE
Document or remove `og:` attributes

### ISSUE THREAD

**Issue by benmccann** [bluwy/svelte-preprocess-import-assets#25]
OpenGraph URLs cannot be relative: https://stackoverflow.com/questions/9858577/open-graph-can-resolve-relative-url

As far as I can tell Vite doesn't allow you to import a fully qualified URL: https://vitejs.dev/guide/assets.html

So I'm not really sure how these could be used. I think you'd probably need to do something a bit more manual and don't really expect you could use a preprocessor for this

https://github.com/bluwy/svelte-preprocess-import-assets/blob/4705fb2d6f5dbd0d6df0bb054d29d9d8751fc919/src/constants.js#L41

**Comment by bluwy** [comment]
Good catch. Yeah I guess converting those as imports wouldn't be useful, but maybe it does open an opportunity for bundlers or setups that can resolve to a full url path?

Today if they use relative paths for those in Vite, even if we preprocess it or not, it would fail so there doesn't seem any harm to continue preprocessing it. We can document it, but I think the behaviour is fine as is today 🤔

**Comment by benmccann** [comment]
I don't know. How would a bundler know which URLs to convert to full URLs?

The reason I think it's confusing to have is that the docs say "See DEFAULT_SOURCES in [src/constants.js](https://github.com/bluwy/svelte-preprocess-import-assets/blob/master/src/constants.js)" and then when you click in there it's a massive list of stuff that's pretty hard to understand. I guess if there was at least some JSDoc about these then it'd save some confusion

**Comment by bluwy** [comment]
The user could have `./asset.png?full` to get the full URL perhaps. Some bundlers or framework may have the `site` information to construct a full URL.

Honestly the "See DEFAULT_SOURCES" part is me being lazy 😄 But I did link to https://github.com/webpack-contrib/html-loader#sources which have the full explanation. I guess we can copy it down here (with credits) so it can be easily referenced.
